pkg/executables/kubectl_test.go

```diff
@@ -429,7 +430,7 @@ func TestKubectlWaitForServiceWithLoadBalancer(t *testing.T) {
 
 func TestKubectlWaitForServiceTimedOut(t *testing.T) {
 	k, ctx, _, e := newKubectl(t)
-	expectedParam := []string{"get", "--ignore-not-found", "--namespace", "eksa-packages", "-o", "json", "--kubeconfig", "kubeconfig", "service", "test"}
+	expectedParam := []string{"get", "--ignore-not-found", "-o", "json", "--kubeconfig", "kubeconfig", "service", "--namespace", "eksa-packages", "test"}
```

Comment:
Why was the name on all of these reorganized?

Reply:
It's because I did a small refactor in kubectl get function to make namespace an optional arg for reusability